src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,33 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin active($_property, $_toValue) {
+  transition: #{$_property} 0.3s;
+  &:active {
+    #{$_property}: $_toValue;
+  }
+}
+
+@mixin focus($_property, $_toValue) {
+  transition: #{$_property} 0.3s;
+  &:focus {
+    #{$_property}: $_toValue;
+  }
+}
+
+@mixin onMiddleScreen {
+  @media (min-width: 640px) {
```

Comment:
Large free space on the right side for tablet view (iPad mini - exists in [Figma](https://www.figma.com/file/Fz588JKGuPS2Bk21De4KE5/Brand-of-eco-cosmetics-_FE-students?node-id=0%3A1) layout)

![image](https://user-images.githubusercontent.com/86923089/191074713-495f0cbb-505a-4090-9bfd-788df8bbad92.png)

Reply:
I can't reproduce this bug( both on the localhost and on GitHub everything is ok
![image](https://user-images.githubusercontent.com/98764641/191204589-ecabdf1e-b216-47df-9376-8996649f3e7e.png)
![image](https://user-images.githubusercontent.com/98764641/191204677-4f8019a4-e770-45aa-b0e6-09ddb7bdc945.png)